Doc/library/collections.abc.rst

```diff
@@ -124,59 +124,70 @@ ABC                            Inherits from          Abstract Methods        Mi
 :class:`Hashable` [1]_                                ``__hash__``
 :class:`Iterable` [1]_ [2]_                           ``__iter__``
 :class:`Iterator` [1]_         :class:`Iterable`      ``__next__``            ``__iter__``
-:class:`Reversible` [1]_       :class:`Iterable`      ``__reversed__``
-:class:`Generator`  [1]_       :class:`Iterator`      ``send``, ``throw``     ``close``, ``__iter__``, ``__next__``
+:class:`Reversible` [1]_       :class:`Iterable`      ``__reversed__``,
+                                                      ``__iter__``
+
+:class:`Generator`  [1]_       :class:`Iterator`      ``send``,               ``close``, ``__iter__``, ``__next__``
+                                                      ``throw``
+
 :class:`Sized`  [1]_                                  ``__len__``
 :class:`Callable`  [1]_                               ``__call__``
 :class:`Collection`  [1]_      :class:`Sized`,        ``__contains__``,
                                :class:`Iterable`,     ``__iter__``,
                                :class:`Container`     ``__len__``
 
 :class:`Sequence`              :class:`Reversible`,   ``__getitem__``,        ``__contains__``, ``__iter__``, ``__reversed__``,
-                               :class:`Collection`    ``__len__``             ``index``, and ``count``
+                               :class:`Collection`    ``__len__``             ``index``, ``count``
 
-:class:`MutableSequence`       :class:`Sequence`      ``__getitem__``,        Inherited :class:`Sequence` methods and
+:class:`MutableSequence`       :class:`Sequence`      ``__getitem__``,        Inherited mixin :class:`Sequence` methods,
```

Comment:
What is the benefit of adding the word `mixin` here?

A downside is that readers may wonder if `mixin methods` or `inherited mixin methods` are a special thing, which they aren’t.  So I would not add the word, `inherited Sequence methods` is fine.

Reply:
It is to refer only to mixing methods in the parent class, as the other parent methods do not belong to this column.